<ui version="4.0" >
 <author>Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net></author>
 <comment>This file is part of the KDE project.
Copyright (C) 2006 Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net>

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
</comment>
 <exportmacro></exportmacro>
 <class>KSpread::SpecialPasteWidget</class>
 <widget class="QWidget" name="SpecialPasteWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Content</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="everythingButton" >
        <property name="text" >
         <string>Everything</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="textButton" >
        <property name="text" >
         <string>Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="formatButton" >
        <property name="text" >
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="commentButton" >
        <property name="text" >
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="resultButton" >
        <property name="text" >
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noBorderButton" >
        <property name="text" >
         <string>Everything without border</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Operation</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="overwriteButton" >
        <property name="text" >
         <string>Overwrite</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="additionButton" >
        <property name="text" >
         <string>Addition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="substractionButton" >
        <property name="text" >
         <string>Substraction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="multiplicationButton" >
        <property name="text" >
         <string>Multiplication</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="divisionButton" >
        <property name="text" >
         <string>Division</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="global" >kdialog.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
